### PR TITLE
docs: Deprecate introduction pages

### PR DESCRIPTION
**Pull request by cpcramer**
**What changed? Why?**
Deprecate all introduction pages. 

- Delete all introduction doc pages.
- Update Get Started links to point towards the components first sub-component.
- Component titles are now all unclickable. 

The current state of our introduction pages are not very valuable - they restate information found in the sidebar and in the individual sub-component pages. We have decided to deprecate all introduction pages and instead write a series of `Guides`. These `Guides` will be step by step instructions on how to actually build things using our components.

The `Guide` section can be used for a more step by step lessons on how to use components and at `Module` level we just have clean docs.

**Notes to reviewers**

**How has it been tested?**
Screenshot 2024-07-12 at 2 16 49 PM

### CODE REVIEW

File: site/docs/pages/getting-started.mdx [hunk 1]
```diff
@@ -227,8 +227,8 @@ For `tailwindcss` users, follow the [Tailwindcss Integration Guide](/guides/tail
 
 Get started with ready-to-use onchain components.
 
-- [**`Identity`**](/identity/introduction) - Display ENS [avatars](/identity/avatar), Attestation [badges](/identity/badge), ENS [names](/identity/name) and account [addresses](/identity/address).
+- [**`Identity`**](/identity/address) - Display ENS [avatars](/identity/avatar), Attestation [badges](/identity/badge), ENS [names](/identity/name) and account [addresses](/identity/address).
```
Comment (Zizzamia) [inline site/docs/pages/getting-started.mdx:230]: I think Identity should land to https://onchainkit.xyz/identity/identity

File: site/docs/pages/index.mdx [hunk 2]
```diff
@@ -107,7 +107,7 @@ bun add @coinbase/onchainkit
     <p className="text-base md:text-base pb-[24px] md:text-left text-center">
       Display ENS [avatars](/identity/avatar), Attestation [badges](/identity/badge), ENS [names](/identity/name) and account [addresses](/identity/address).
     </p>
-    <a href="/identity/introduction" className="mx-auto md:m-0 flex justify-center items-center border border-indigo-400 text-indigo-400 font-bold leading-6 px-4 py-3 rounded-xl border-solid w-[180px]">
+    <a href="/identity/address" className="mx-auto md:m-0 flex justify-center items-center border border-indigo-400 text-indigo-400 font-bold leading-6 px-4 py-3 rounded-xl border-solid w-[180px]">
```
Comment (Zizzamia) [inline site/docs/pages/index.mdx:110]: To https://onchainkit.xyz/identity/identity

File: site/docs/pages/xmtp/introduction.mdx [hunk 3]
```diff
@@ -1,67 +0,0 @@
----
-title: Introduction to XMTP utilities · OnchainKit
```
Comment (Zizzamia) [inline site/docs/pages/xmtp/introduction.mdx:2]: So this intro actually is kind of useful..... So we might want to move this content somewhere else.

Comment (Zizzamia) [reply site/docs/pages/xmtp/introduction.mdx:2]: Or we need to improve the docs on the XMTP pages.

Comment (cpcramer) [reply site/docs/pages/xmtp/introduction.mdx]: Let's leave the `XMTP introduction` page for now until we have a `XMTP Guide` that will include this important information and ultimately replace this page.

**Review by Zizzamia — APPROVED** [review]

**Pull request merged**